winbuild/Makefile.vc

```diff
@@ -118,6 +114,10 @@ USE_MBEDTLS = true
 MBEDTLS     = $(WITH_MBEDTLS)
 !ENDIF
 
+!IF ( "$(USE_SSL)"=="true" && "$(USE_WINSSL)"=="true" ) || ( "$(USE_SSL)"=="true" && "$(USE_MBEDTLS)"=="true" ) || ( "$(USE_MBEDTLS)"=="true" && "$(USE_WINSSL)"=="true" )
```

Comment:
That's a pretty long line, I suggest you wrap it like so:
```
!IF "$(USE_SSL)"=="true" && "$(USE_WINSSL)"=="true" \
 || "$(USE_SSL)"=="true" && "$(USE_MBEDTLS)"=="true" \
 || "$(USE_MBEDTLS)"=="true" && "$(USE_WINSSL)"=="true"
```
Otherwise well done!